packages/volto/theme/themes/pastanaga/extras/blocks.less

```diff
@@ -275,6 +275,58 @@ body.has-toolbar.has-sidebar-collapsed .ui.wrapper > .ui.inner.block.full {
   }
 }
 
+// Hacks for keeping the full-width image in the edit screen
+.view-editview {
+  &.has-sidebar {
+    .block.align.full img {
+      width: calc(100vw - 375px) !important;
```

Comment:
@cihanandac We can't do this. If you change the width in edit view. How will a user know its applied a full width, which means the viewport width? Ideally in Volto we wish to keep Edit and View display components the same. For eg: https://github.com/plone/volto/blob/4e5ecfde9f5c65a1b5f2d70dd2dd1bc7d5749f77/packages/volto/src/components/manage/Blocks/Listing/Edit.jsx#L39